c-api/memory.po

```diff
@@ -54,6 +64,13 @@ msgid ""
 "internal buffers is performed on demand by the Python memory manager through "
 "the Python/C API functions listed in this document."
 msgstr ""
+"Il est important de comprendre que la gestion du tas en Python est effectuée "
+"par l'Interpréteur lui même et que l'utilisateur n'a pas de contrôle dessus, "
+"même s'ils manipulent des objets pointeurs sur les blocs de mémoire à "
```

Comment:
La version originale n'est pas claire, elle passe de « the user » à « they ».
En français, ça me fait tiquer.